l10n-sample.xlf

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is the XLF file generated from the bundle.l10n.json file by using command:
npx @vscode/l10n-dev generate-xlf -o ./l10n-sample.xlf ./l10n/bundle.l10n.json ./package.nls.json

We can start generate this XLF file in place of our existing XLF file at "loc\translations-export\vscode-powerplatform.xlf"